src/plugins/intel_cpu/src/ngraph_transformations/op/ngram.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <openvino/core/node.hpp>
+#include <openvino/op/op.hpp>
+
+namespace ov {
+namespace intel_cpu {
+/**
+ * The operation flattens embedding tensor of token vectors and traverse it by a sliding window of size times the original embedding sizes.
+ * Inputs:
```

Comment:
Please add allowed data types for Inputs/Outputs and classifier Required/Optional for Inputs.

Reply:
Added, thanks